countries/views/delete.erb

```diff
@@ -8,6 +8,7 @@
         </ul>
         <br>
         <form action="/questions/delete" method="post">
+            <input type = "hidden" name = "id" value = <%= @question.id %>>
```

Comment:
It is convention for html attributes to not have spaces in between the `=` sign, example: `type="hidden"`